packages/bpk-component-image/src/withLazyLoading.tsx

```diff
@@ -130,7 +114,9 @@ export default function withLazyLoading(
             return supportsPassiveOption;
           },
         });
+        // @ts-expect-error TODO: make this TS compliant
```

Comment:
changing this causes the unit tests to fail so it needs a bit of a bigger investigation. Added a ts exception for now

Reply:
Maybe we should check this as part of this work as it means that something must be wrong and we don't want to risk releasing a component that has a bug into it after we move to TS

Reply:
Let's have a chat about this one as I do worry about us introducting a bug by doing workarounds and doesn't follow great coding practice to work around it to make tests just pass

Reply:
It's not introducing a bug though, at most there's already a bug present which we're not fixing in this PR 😞 but yes, let's have a chat!

Reply:
ok I fixed it 😅 turns out that it would always throw an error because of the args in `addEventListener`/`removeEventListener`!